root/static/scripts/guess-case/MB/Control/GuessCase.js

```diff
@@ -16,7 +16,7 @@ import gc from '../GuessCase/Main';
 import * as modes from '../../modes';
 import MB from '../../../common/MB';
 
-MB.Control.initialize_guess_case = function (type, formPrefix) {
+MB.Control.initializeGuessCase = function (type, formPrefix) {
```

Comment:
There are also some cases in our *.tt files where this needs to be changed now - see `git grep initialize_guess_case` for a complete list.

Reply:
I have made the chagnes.

Reply:
I have made the changes